.github/workflows/ts_tests.yml

```
@@ -0,0 +1,22 @@
+name: Tree-sitter test
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  test:
+    name: Tree-sitter grammar tests
+    runs-on: ubuntu-latest
+    steps:
+        - name: Checkout repository
+          uses: actions/checkout@v4
+
+        - run: rustup toolchain install stable
+
+        - name: Install tree-sitter cli
+          run: cargo install tree-sitter-cli
```

Same here. Using this action should use cargo-binstall which should be faster than compiling the cli ourselves.
```suggestion
        - name: Install tree-sitter CLI
          uses: taiki-e/install-action@v2
          with:
            tool: tree-sitter-cli
```

---

I'm still kinda new to github workflows and wasn't aware of this. Seems very cool!